src/use_case/login/LoginUserDataAccessInterface.java

```diff
@@ -0,0 +1,11 @@
+package use_case.login;
+
+import entity.User;
+
+public interface LoginUserDataAccessInterface {
+    boolean existsByName(String identifier);
+
+    void save(User user);
```

Comment:
Why do you need the save method for your use case?

Reply:
Removed in the next version.